DATA_CHANGELOG.md

```diff
@@ -0,0 +1,10 @@
+# Data model Changelog
```

Comment:
Undecided as to whether root is the best location for this to live vs. where the models live.. 🤔 Better for visibility, of course..

Reply:
I think `DATA_MODEL_CHANGELOG` is a better name, despite the length, as `DATA_CHANGELOG` is rather vague.

Reply:
Yeah, I opted to put it in the root for visibility, especially the CHANGELOG. But happy to move it if that seems better

Reply:
Having had the weekend to think on it, I'm OK with root. 👍